express/blocks/sticky-promo-bar/sticky-promo-bar.css

```diff
@@ -33,6 +32,18 @@ main .sticky-promo-bar a:any-link {
     margin-top: 5px;
 }
 
+main .sticky-promo-bar.rounded a:any-link {
+    color: white;
+    text-decoration: underline;
```

Comment:
nit: I might add a `white-space: nowrap;` just to prevent something like this:

<img width="478" alt="Screen Shot 2023-06-12 at 4 21 14 PM" src="https://github.com/adobe/express-website/assets/57737624/87243c68-7e9d-4301-a1fe-aa13056c871f">